src/policy-server/cc_client/client.go

```diff
@@ -286,14 +292,19 @@ func (c *Client) GetSubjectSpaces(token, subjectId string) (map[string]struct{},
 
 	route := fmt.Sprintf("/v2/users/%s/spaces", subjectId)
```

Comment:
Can we default `results-per-page` to 100 (the maximum)? That will cut the number of requests in half.